src/components/list-with-actions/_list-with-actions.scss

```diff
@@ -4,22 +4,46 @@
 
 .hmrc-list-with-actions {
   @include govuk-font($size: 19);
+
+  .govuk-\!-font-weight-regular {
+    font-weight: normal !important;
+  }
+
+  @include govuk-media-query($from: desktop) {
+    .govuk-\!-summary-list-key-width {
+      width: 50% !important;
+    }
+
+    .govuk-\!-summary-list-actions-width {
+      width: 50% !important;
+    }
+  }
 }
 
-// Kept for backward compatibility with add-to-a-list
-.hmrc-list-with-actions--short {
+.hmrc-list-with-actions--long {
   @include govuk-media-query($from: desktop) {
+    .govuk-\!-summary-list-key-width {
+      width: 30% !important;
+    }
+
+    .govuk-\!-summary-list-actions-width {
+      width: 70% !important;
+    }
+
+    // Kept for backward compatibility with add-to-a-list
     .hmrc-list-with-actions__name {
-      width: 30%;
+      width: 50%;
```

Comment:
Again, we can remove this class and just use an override class from GOV.UK

Reply:
We are not using the hmrc-list-with-actions__name css style in the template anymore but leaving it here for backward compatibility.